src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,74 @@
+package study;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+
+	@Test
+	@DisplayName("문자열을 split 하면 문자열 배열을 반환한다.")
+	void Given_String_When_Split_Then_SplitToArr(){
+		// given
+		String str = "1,2";
+
+		// when
+		String[] result = str.split(",");
```

Comment:
```suggestion
        final String[] actual = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?